src/main/resources/com/datapipe/jenkins/vault/configuration/VaultConfiguration/config.jelly

```diff
@@ -4,7 +4,10 @@
     <f:entry title="Vault URL">
         <f:textbox field="vaultUrl" name="vaultUrl"/>
     </f:entry>
-    <f:entry title="Vault Credential" field="vaultCredentialId">
+    <f:entry title="Vault CA Certificate">
+        <f:textarea field="vaultCACert" name="vaultCACert"/>
```

Comment:
I suggest putting it under advanced.
Also best depend on multiline-secrets-ui
```
<dependency>
  <!-- TODO: after upgrading jenkins.version >= 2.171, migrate dependency -->
  <groupId>io.jenkins.temp.jelly</groupId>
  <artifactId>multiline-secrets-ui</artifactId>
  <version>1.0</version>
</dependency>
```

It should look like this:
```xml
<f:entry title="Vault CA Certificate" field="vaultCACert">
  <secretTextarea xmlns="/io/jenkins/temp/jelly"/>
</f:entry>
```